### ISSUE TITLE
Not compatible with valheim plus

### ISSUE THREAD

**Issue by thordomr** [blaxxun-boop/PassivePowers#6]
I'm getting the error with the latest version saying it's disabled because it's not compatible with valheim plus

**Comment by SephirothFF7R** [comment]
This is the case with all of his mods going forward. You should look into something to replace V+ with as it is less reliable than other mods that do the same thing; hence the reason both he and many mod authors are pulling their mods support for V+.

**Comment by LegalSmeagol** [comment]
Yeah, as @SephirothFF7R said, this is intended and will probably be the case for a lot of other mods too. I would refer to his response here: https://github.com/blaxxun-boop/PassivePowers/issues/5 which is pretty sound and reasonable imo. He also gives a pretty solid explanation as to why an all-in-1 mod like V+ is bad.

**Comment by SerpentDrago** [comment]
so far on my quest to replace v+ , i'm looking at needing around 10 mods .. so far and i'm not even done yet . minor things like endless torches / expanded workbench area / bigger chests / bigger pickup radius / auto equip shield with primary / no un equiping when in water / etc etc .

This is an absolute pain in the ass . Just saying . Love your addon , wish you could keep the fight up but i do understand . I may end up choosing to have to remove your mod for now . Its going to take me weeks of work to switch over / share config files for minor addons that don't have server side configs ... etc etc . 

Really wish you would rethink

**Comment by SephirothFF7R** [comment]
> so far on my quest to replace v+ , i'm looking at needing around 10 mods .. so far and i'm not even done yet . minor things like endless torches / expanded workbench area / bigger chests / bigger pickup radius / auto equip shield with primary / no un equiping when in water / etc etc .
> 
> This is an absolute pain in the ass . Just saying . Love your addon , wish you could keep the fight up but i do understand . I may end up choosing to have to remove your mod for now . Its going to take me weeks of work to switch over / share config files for minor addons that don't have server side configs ... etc etc .
> 
> Really wish you would rethink

Really? Took me a day. Are you using r2modmanager at all? It can easily share mods and configs with others via a code you can share and it will download and configure all the mods for them automatically.

**Comment by SerpentDrago** [comment]
yes i use r2modmanager , i have a modpack all users on my server use , it pushes configs and mods to them (doesn't have the ability to Uninstall mods though ) 

I've been going though my Vplus config file line by line at all the things i use . and  then trying to find a mod alternative . its a DAUNTING list of things . to the point , i'm thinking of just giving up on converting . I've spent hours already trying to organize my game plane without loosing things in the game . or loosing features . 

I hate that i relied on Vplus so much , I get the issues with it  . but this is going to be hard to impossible.

some of the things i have to find replacements and if not server side config push out config files  >  Expanded Chests / expanded stack sizes / autokiln/smelter / no fuel used for torches / expand workbench area / auto repair items with workbench/crafting table/map exploring share / weight reduction/pickupradiius / auto honey collection/ weather damage/ repair in area of effect / expand comfort radius / the list goes on and on 

some of the replacements i hope to work > and weather they support server side config / client only : > Bigger Chests C&S / GracefulLanding C only / AutomaticFuel C&S / AzuWorkbench inventory repair C only / ItemStacksandWeight C only config share / CraftingStationRange C only config share / BiggerPickupRadius same as last/ TorchesAndResin C only

**Comment by SerpentDrago** [comment]
Honestly it was already a bit of work to keep up with .. this is prob going to push me over the edge , i hope not to many devs follow suit . i just want to play the game , and vplus made it extreamly easy (to easy) to easly push configs to clients via a simple config on the server side )

**Comment by blaxxun-boop** [comment]
https://github.com/blaxxun-boop/PassivePowers/issues/5
